aws/internal/service/macie2/waiter/waiter.go

```diff
@@ -0,0 +1,32 @@
+package waiter
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const (
+	// Maximum amount of time to wait for an Member to return NotFound
+	MemeberNotFoundTimeout = 5 * time.Minute
```

Comment:
Since we're waiting for the member relationship to be `Invited`, this should be something like

```suggestion
	// Maximum amount of time to wait for the MemberRelationshipStatus to be Invited, Enabled, or Paused
	MemberInvitedTimeout = 5 * time.Minute
```